core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataManagerTest.scala

```diff
@@ -640,8 +640,14 @@ class GroupMetadataManagerTest {
     val offsetCommitRecords = createCommittedOffsetRecords(committedOffsets)
     val memberId = "98098230493"
     val groupMetadataRecord = buildStableGroupRecordWithMember(generation, protocolType, protocol, memberId)
+
+    // Should ignore unknown record
```

Comment:
I was thinking about this one. I wonder if it would not be better to pull this into a separate unit test instead of hijacking existing ones. Is it possible?